Documentation/specs/runtime-configuration-file.md

```diff
@@ -10,21 +10,25 @@ There are two runtime configuration files for a particular application. Given a
 
 * `MyApp.dll` - The managed assembly for `MyApp`, including an ECMA-compliant entry point token.
 * `MyApp.exe` - A copy of the `corehost.exe` executable.
-* `MyApp.config.json` - An **optional** configuration file containing runtime configuration settings.
+* `MyApp.runtimeconfig.json` - An **optional** configuration file containing runtime configuration settings.
```

Comment:
Because the `.runtimeconfig.json` file is designed to be user-editable (much like `[assemblyname].exe.config` in full framework) whereas `.deps.json` is not. I'll clarify that.